components/blitz/src/omero/gateway/facility/ROIFacility.java

```diff
@@ -43,7 +43,6 @@
 import omero.gateway.model.ROIResult;
 import omero.gateway.util.ModelMapper;
 import omero.gateway.util.PyTablesUtils;
-import omero.model.IObject;
```

Comment:
Yes, I'll restore the imports to match the `regions` ones. I just noticed that I also should remove the `PyTablesUtils` and `TableData` classes. Not sure how they slipped in there, these classes should have been part of the `TablesFacility` commit, which I did not cherry-pick for this PR.